docs/build/build-protocol-info.md

```diff
@@ -302,6 +302,16 @@ Although upgrading your nodes is generally not necessary to follow an upgrade, w
 following the Polkadot releases and upgrading in a timely manner, especially for high priority or
 critical releases.
 
+### Transaction Version Upgrades
+Apart the `runtime_version` there is also the `transaction_version` which denotes how to correctly encode/decode calls for a given runtime (useful for hardware wallets). The reason `transaction_version` is separate from `runtime_version` is that it explicitly notes that the call interface is broken/not compatible. Hence, a signed transaction blob will be rejected by an incompatible runtime, rather than being misinterpreted.
```

Comment:
> Hence, a signed transaction blob will be rejected by an incompatible runtime, rather than being misinterpreted.

Not quite. It's true but a separate problem. The call that you construct may in fact be misinterpreted. I would just delete this sentence.